beams/beams/doctype/job_description_template/job_description_template.json

```diff
@@ -0,0 +1,61 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "autoname": "field:job_description_template",
+ "creation": "2024-10-02 09:30:23.177786",
+ "doctype": "DocType",
+ "engine": "InnoDB",
+ "field_order": [
+  "job_description_template",
+  "description",
+  "designation"
+ ],
+ "fields": [
+  {
+   "fieldname": "job_description_template",
```

Comment:
field names should be intuitive, change it to something like "Template Name" or "Job Description Template Name"